lib/widgets/user.dart

```diff
@@ -1,5 +1,4 @@
 import 'package:flutter/material.dart';
-
```

Comment:
Please don't delete this blank line; it separates third-party imports from our own imports. (Here and elsewhere in the PR.)